### PR TITLE
Testing with aiodocker

### PR DESCRIPTION
**Pull request by achimnol**
refs aio-libs/aiodocker#21.

### CODE REVIEW

**Comment by jettify** [comment]
Looks nice! Will also try locally tonight.
Travis complains about docker version. Is there a way to specify client version in `aiodocker`?
```
aiodocker.exceptions.DockerError: DockerError(400, 'client is newer than server (client API version: 1.26, server API version: 1.24)')
```

**Comment by achimnol** [comment]
@jettify Not yet, but it's possible to upgrade the default docker daemon provided in Travis CI. I think API v1.24 and v1.26 are almost compatible, as Docker increments the API version upon nearly every release with minor/no changes in existing APIs.

File: setup.py [hunk 1]
```diff
@@ -8,8 +8,8 @@
 
 PY_VER = sys.version_info
 
-if not PY_VER >= (3, 5):
-    raise RuntimeError("aioodbc doesn't support Python earlier than 3.5")
+if not PY_VER >= (3, 6):
```
Comment (jettify) [inline setup.py]: I do not think this necessary, 3.5 works fine, we just test library with 3.6 on each PR

**Comment by jettify** [comment]
No big difference for us, newer better

File: tests/conftest.py [hunk 2]
```diff
@@ -20,69 +20,61 @@ def session_id():
...
+        asyncio.set_event_loop_policy(uvloop.EventLoopPolicy())
+    loop = asyncio.get_event_loop_policy().new_event_loop()
     yield loop
     gc.collect()
     loop.close()
-    asyncio.set_event_loop(old_loop)
 
 
+@pytest.fixture(scope='session')
+async def docker(event_loop):
```
Comment (jettify) [inline tests/conftest.py]: There is not reason to rename loop everywhere, just create alias
```
@pytest.fixture
def loop(event_loop):
   return event_loop
```

**Comment by achimnol** [comment]
Just a note: feel free to "squash" all my dirty commits. ;)

File: setup.py [hunk 3]
```diff
@@ -56,6 +57,7 @@ def read_version():
       download_url='https://pypi.python.org/pypi/aioodbc',
       license='Apache 2',
       packages=find_packages(),
+      python_requires='>=3.5',
```
Comment (jettify) [inline setup.py]: did not know about this option, thanks!

**Comment by jettify** [comment]
Good job! Thanks a lot!

**Pull request merged**